concourse/pipeline.yml

```diff
@@ -105,10 +105,12 @@ jobs:
             args:
               - -c
               - |
-                set -ue
-                cat .git/committer > ../committer-details/last-committer
-                postgres -p 5432 -D /usr/local/var/postgres >&1 &
-                bundle exec rake
+                set -eu
+                service postgresql start
+                su - postgres -c "psql -c \"create role root with createdb login password 'password';\""
+                DATABASE_URL="postgres://root:password@localhost:5432/coronavirus_form_development" bundle exec rails db:setup
```

Comment:
We can set this once in `params` before the `run`:

```yaml
params:
        DATABASE_URL: "postgres://root:password@localhost:5432/coronavirus_form_development
```

Reply:
What does that do? Does that set it as a env variable for everything in run?
Or does it prefix it as a parameter to all those calls?

Reply:
The former.

Reply:
Does job have a `params:` ? just been reading up [in the docs](https://concourse-ci.org/jobs.html#schema.job) I can see params in the steps schema, but not in jobs :thinking:

Reply:
Oh! Sorry. 😩

Reply:
It might well do, not actually tested it, and i find these docs well confusing.
Was wondering if you'd seen it used somewhere before?